screens/TabTwoScreen.tsx

```diff
@@ -1,39 +1,59 @@
-const URLshatin = 'https://api.data.gov.hk/v1/nearest-schools?lat=22.386140737643018&long=114.18949127197266'
-const URLtw = 'https://api.data.gov.hk/v1/nearest-schools?lat=22.366839471993924&long=114.11275863647461'
-const URLkwaitsing = 'https://api.data.gov.hk/v1/nearest-schools?lat=22.330005374001328&long=114.11138534545898'
-const URLsaikung = 'https://api.data.gov.hk/v1/nearest-schools?lat=22.28553747295002&long=114.27377700805664'
-const URLkt = 'https://api.data.gov.hk/v1/nearest-schools?lat=22.316984097908534&long=114.22708511352539'
-const URLyautsummong = 'https://api.data.gov.hk/v1/nearest-schools?lat=22.297799818524283&long=114.1749858856201'
-const URLssp = 'https://api.data.gov.hk/v1/nearest-schools?lat=22.335055505656236&long=114.15915012359619'
-const URLklc = 'https://api.data.gov.hk/v1/nearest-schools?lat=22.328545255191877&long=114.18756008148193'
+//18districts api link
+const URLwongtaisin =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.34105761010081&long=114.19532775878906'
+const URLsouthern =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.2375920813679&long=114.1552448272705'
+const URLwestern =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.28320326161826&long=114.13880825042723'
+const URLwanchai =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.27951820593017&long=114.17861223220824'
+const URLeastern =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.285569737161087&long=114.21996116638184'
+const URLisland =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.27371834339708&long=113.91895294189453'
+const URLtuenmun =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.388045420568076&long=113.95671844482422'
+const URLYuenLong =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.443699987801786&long=114.0283441543579'
+const URLnorthern =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.498219584753013&long=114.1398811340332'
+const URLtaipo =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.447491635933645&long=114.16623115539551'
+const URLshatin =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.386140737643018&long=114.18949127197266'
+const URLtw =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.366839471993924&long=114.11275863647461'
+const URLkwaitsing =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.330005374001328&long=114.11138534545898'
+const URLsaikung =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.28553747295002&long=114.27377700805664'
+const URLkt =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.316984097908534&long=114.22708511352539'
+const URLyautsummong =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.297799818524283&long=114.1749858856201'
+const URLssp =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.335055505656236&long=114.15915012359619'
+const URLklc =
+  'https://api.data.gov.hk/v1/nearest-schools?lat=22.328545255191877&long=114.18756008148193'
```

Comment:
Nit: Use of Enum that could group related constants together for easier future references.